### ISSUE TITLE
Can we drop the spec version?

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/pointerlock#83]
As the W3C process allows for "living standards" once specs reach REC, it's probably not longer necessary to have a the "2.0" version number any more. 

Could we drop that or is there some technical reason for keeping it? (e.g., V2.0 has breaking changes from V1.0)

**Comment by mustaqahmed** [comment]
Good point, I don't see any blocker to switching to a "living standard".  I will look into this if there is not objection to this proposal in the next ~2 weeks.
